packages/tradershub/src/components/CurrencySwitcher/CurrencySwitcher.tsx

```diff
@@ -22,6 +25,13 @@ const AccountActionButton = ({ balance, isDemo }: AccountActionButtonProps) => {
         <Button
             className='flex items-center justify-center border-solid h-1600 py-300 px-800 rounded-200 border-sm border-system-light-less-prominent-text'
             colorStyle='black'
+            onClick={() => {
+                if (isDemo) {
```

Comment:
Shouldn't it be the same as the condition as the one at line 18?

Reply:
We wont be seeing the reset balance button anyway if its 10000